### PR TITLE
[IntentsUI] Update framework to Xcode 12 beta 4.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Comment by mandel-macaque** [comment]
@dalexsoto wont, I run it before I do the PR

**Comment by dalexsoto** [comment]
The real problem is that IntentsUI is not enabled in tvOS ;)

**Comment by mandel-macaque** [comment]
Wait,cause xtro does not complain about it.

**Comment by dalexsoto** [comment]
Yeah my bad, intro should complain because IntentsUI is not enabled in tvOS so those attributes are likely doing nothing for tvOS

File: tests/xtro-sharpie/tvOS-IntentsUI.ignore [hunk 1]
```diff
@@ -1,6 +1,9 @@
+## not exposed by our API (better use the OS version)
 !missing-field! IntentsUIVersionNumber not bound
 !missing-field! IntentsUIVersionString not bound
+
+# fails on instro rdar: https://github.com/xamarin/maccore/issues/2286
```
Comment (spouliot) [inline tests/xtro-sharpie/tvOS-IntentsUI.ignore:5]: typo `intro`

Comment (mandel-macaque) [inline tests/xtro-sharpie/tvOS-IntentsUI.ignore:5]:
```suggestion
# fails on intro rdar: https://github.com/xamarin/maccore/issues/2286
```

**Comment by mandel-macaque** [comment]
@spouliot it does not and should not be added. The diff only adds the selectors that do not respond as per intro checks. So there is nothing to do except ignore them.

**Comment by spouliot** [comment]
@mandel-macaque if the framework is not linked/loaded then the selectors will fail

**Comment by spouliot** [comment]
simulator tvOS 14
```
...
[FAIL] iOSApiTypoTest.ApiTypoTest.ConstantsCheck : IntentsUILibrary 
```
so the library does not exists, that's likely the root issue.


device tvOS 14 - an unrelated crash on my old AppleTV
```
21:06:37.3524400 21:06:37.3524080 /Library/Caches/com.apple.xbs/Sources/MetalImage/MetalImage-124.0.20/MPSCore/Utility/MPSPredicate.mm:102: failed assertion [MPSPredicate initWithBuffer:offset:] device: Apple A8 GPU does not support predication.’
```
I'll re-run it without `MPSPredicate` (and there might be others)

**Comment by spouliot** [comment]
`IntentsUILibrary` is not on tvOS 14 devices either (at least not the older, non-4k, devices)

**Pull request merged**